src/controller/java/templates/ChipIdLookup-java.zapt

```diff
@@ -52,5 +52,98 @@ public final class ChipIdLookup {
     {{/all_user_clusters}}
     return "";
   }
+
+  {{#chip_client_clusters}}
+  public static final class {{asUpperCamelCase name}}Cluster {
```

Comment:
This is a stand-alone class. Could you please move it into aseparate codegen so that we have individual generations?

Also please move it to jinja templates. I would prefer for java to standardize on jinja instead of zap.

Reply:
Ideally we would have separate java files for each cluster as well (jinja templates allow for this)

Reply:
Yes, I will review if it is possible with jinja.

Reply:
@andy31415 
jinja file was created. Is there a separate script used for creation?
It seems that build errors occur in some cases after uploading the file.

Reply:
@andy31415 
Do you know how to test after creating a jinja file?
If you have any related documents, it would be helpful if you could share them.

Reply:
I don't know how to solve below error log.

`======================================================================
FAIL: test_generators (__main__.TestGenerators) (idl='inputs/several_clusters.matter', generator='java-class')
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/__w/connectedhomeip/connectedhomeip/out/coverage/../../scripts/py_matter_idl/matter_idl/test_generators.py", line 144, in run_test_cases
    generator.render(dry_run=False)
  File "/__w/connectedhomeip/connectedhomeip/scripts/py_matter_idl/matter_idl/generators/__init__.py", line 142, in render
    self.internal_render_all()
  File "/__w/connectedhomeip/connectedhomeip/scripts/py_matter_idl/matter_idl/generators/java/__init__.py", line 678, in internal_render_all
    self.internal_render_one_output(
  File "/__w/connectedhomeip/connectedhomeip/scripts/py_matter_idl/matter_idl/generators/__init__.py", line 181, in internal_render_one_output
    self.storage.write_new_data(output_file_name, rendered)
  File "/__w/connectedhomeip/connectedhomeip/out/coverage/../../scripts/py_matter_idl/matter_idl/test_generators.py", line 99, in write_new_data
    self.checker.assertEqual(
AssertionError: '/*\n[682 chars];\n\n\npublic class ClusterIDMapping {\n  publ[152689 chars]n}\n' != '/*\n[682 chars];\n\nimport java.util.HashMap;\nimport java.ut[6723 chars]}}\n' : Content of java/chip/devicecontroller/ClusterIDMapping.java

----------------------------------------------------------------------`

Reply:
I found a way!

Reply:
@joonhaengHeo  have you figured out how to run and test jinja file?

Reply:
I used "./scripts/codegen.py".
generator : java-class

e.g, : `./scripts/codegen.py --generator java-class ./scripts/py_matter_idl/matter_idl/tests/inputs/several_clusters.matter
`